### DISCUSSION TITLE
How do I disable copilot chat without disabling copilot autocompletions

### DISCUSSION THREAD

**Discussion by Fasteroid** [community/community#130265] [Copilot Conversations] [answered]
This was "answered" before in #68276 but the answer provided no longer works.  

The original copilot is fantastic, and even funny at times, but copilot chat has gotten in my way more than it's helped over the past week.  I'd like to disable it and save some CPU cycles.  Has anyone found a way since microsoft (probably deliberately) removed the disable checkbox?

NOTE: Disabling the copilot chat extension also disables the original copilot extension.  They seem to be interdependent now with no way to only use one.

**Comment by alexisseurin** [comment]


**Reply by Fasteroid** [reply]
> To disable Copilot Chat while keeping Copilot autocomplete enabled, follow these steps:
> 
> 1) Open VS Code and go to Extensions
> 2) Search for "GitHub Copilot Chat"
> 3) Disable the Chat extension by clicking on the gear icon next to it and selecting "Disable"
> 4) Ensure the main Copilot extension is still enabled for autocompletions

I meant to say something in the initial post... this doesn't work—it disables both copilot extensions.  They seem to be interdependent now.

**Comment by alexisseurin** [comment, marked as answer]
I did it! @Fasteroid 

First, install another version of Copilot Chat and select version 0.2.0
![image](https://github.com/community/community/assets/44972852/73afee0f-6dd4-4144-bf86-02ba4e035680)

Next, uninstall Copilot Chat

Finally, restart the extensions

**Reply by Fasteroid** [reply]
What a hack!  I'm kind of amazed that actually works.

**Reply by xiangp126** [reply]
Awesome! It does work!

**Reply by pochtar** [reply]
This really helped, thanks!

**Reply by Try431** [reply]
Can confirm, this doesn't work anymore.

**Reply by Charlesbasis** [reply]
@Try431 right.

**Reply by Kamrulthedev** [reply]
Awesome! It does work!

**Reply by Fasteroid** [reply]
saved me again, thank you 👍

**Comment by socsoft-asd** [comment]
Even if I deleted the github copilot chat, the plugin kept being re-enabled, so it was fixed after changing the Settings > profile.

**Comment by Kamrulthedev** [comment]
Awesome! It does work!

**Comment by Kamrulthedev** [comment]
Awesome! It does work!
